db/migrate/20160408012345_create_pg_search_documents.rb

```diff
@@ -0,0 +1,18 @@
+class CreatePgSearchDocuments < ActiveRecord::Migration
+  def self.up
```

Comment:
This could prob be a def self.change as drop_table is reversible,
but idk how important that is.